libs/filabridge/include/private/filament/EngineEnums.h

```diff
@@ -43,7 +43,7 @@ namespace BindingPoints {
     constexpr uint8_t PER_RENDERABLE          = 1;    // uniforms/samplers updated per renderable
     constexpr uint8_t PER_RENDERABLE_BONES    = 2;    // bones data, per renderable
     constexpr uint8_t LIGHTS                  = 3;    // lights data array
-    constexpr uint8_t POST_PROCESS            = 4;    // samplers for the post process pass
+    constexpr uint8_t POST_PROCESS_DEPRECATED = 4;    // deprecated
```

Comment:
I assume we need to keep it for compatibility with existing materials? We could just call is UNUSED.